translations/sv.po

```diff
@@ -633,3 +633,51 @@ msgstr "Prenumerera på RSS-flöde med nyheter baserat på dina val"
 msgctxt "React search clear selection label"
 msgid "Clear @label selection"
 msgstr "Rensa alla @label"
+
+msgctxt "Service channel type verb"
+msgid "Use the online service"
```

Comment:
Some of these can be moved here https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/tree/main/translations